### ISSUE TITLE
Loadbalancer Backend Limit

### ISSUE THREAD

**Issue by theyough** [scaleway/scaleway-cloud-controller-manager#94]
Hi,
I'm using kubernetes and I would like to create 2000 frontend to 1 backend with ssl-proxy-protocol-v2, i'm issuing a 429 HTTP error, Too many requests

Also, it look like than i can't do that, cause the scaleway loadbalancer create 1 frontend for 1 backend only, In my case i need to create X frontends to 1 backend.

Can you help me ?
Thanks

**Comment by jtherin** [comment]
Hello, I don't understand your usage.
You want to forward 2000 ports to the same nodeport ?

**Comment by theyough** [comment]
Yes exactly, cause our customer can connect to one of 2000 ports, and then we forward the traffic to one backend with the proxy protocol v2.
From the pod behind the backend, we are able to know from with port the client has been connected, then we can apply internally a different configuration settings for his connection.

**Comment by theyough** [comment]
One of other problem, is that i can't create more than 300 frontend or backend right ?

**Comment by theyough** [comment]
@jtherin , from what i see on scaleway website should be unlimited,  Is it software restriction or user restriction that you apply ?

**Comment by jtherin** [comment]
I'm not from loadbalancer team, i'm not aware of limitation of numbers of frontends/backends, for that, better to open a support socket or ask #load-balancer on our slack.

on k8s site, we can implement a throttling to avoid 429, but by design, we require 1 frontend = 1 backend on loadbalancer side, so opening 2000 ports on the loadbalancer will require opening 2000 nodeports (even if forwarding to the same port of the pod)
